main.go

```diff
@@ -43,11 +45,31 @@ func newMetrics(reg prometheus.Registerer) *metrics {
 			Name: "govuk_mirror_last_updated_time",
 			Help: "Last time the mirror was updated",
 		}, []string{"backend"}),
+		mirrorResponseStatusCode: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "govuk_mirror_response_status_code",
+			Help: "Response status code for the availability the mirror",
```

Comment:
optional: missing "of", but how about:
```suggestion
			Help: "Response status code for the MIRROR_AVAILABILITY_URL probe",
```